public/locales/en-US/grafana.json

```diff
@@ -1391,7 +1391,8 @@
       "title": "Profiles"
     },
     "public": {
```

Comment:
Food for thought: Should we take this chance to rename the the translation keys since 'public' and 'public-dashboard' does not make sense anymore or take into account of that when we work on redesign of card components?

Reply:
I think this is something we need to discuss internally. I made new React components with "share externally" naming for better understanding, but we have "public dashboards" and "share externally" terms coexisting at the same time. I would leave it for now

Reply:
I just realized we also need to have conditionals here, as this section will still be "public dashboard" when the FF is not enabled